files/apis/03-06-args.R

```diff
@@ -0,0 +1,4 @@
+#' @post /login
+function(req){
+  cat(req$args$uname, req$args$psw)
+}
```

Comment:
This API could go ahead and host the HTML file, too.

Reply:
Done.

Reply:
Be it using the `@html` tag!